pkg/graphql/execution_engine_test.go

```diff
@@ -271,6 +271,29 @@ func TestExecutionEngine_ExecuteWithOptions(t *testing.T) {
 		}),
 		expectedResponse: `{"data":{"hero":"Human"}}`,
 	}))
+
+	t.Run("execute query and apply input coercion for lists", runWithoutError(testCase{
+		schema: inputCoercionForListSchema(t),
+		request: func(t *testing.T) Request {
+			return Request{
+				OperationName: "charactersByIds",
+				Variables: stringify(map[string]interface{}{
+					"ids": 1,
+				}),
+				Query: `query($ids: [Int]) {charactersByIds(ids: $ids) { name }}`,
```

Comment:
would it make sense to add another test without a variable definition but inline id? e.g. `{charactersByIds(ids: 1) { name }}`

Reply:
I added a test for this query: `query($ids: [Int]) {charactersByIds(ids: 1) { name }}` and removed the variable definition. Now it returns `{ "ids":  }` as `expectedBody`. I found that It's not modified by the input coercion implementation. Is that the expected behavior? 

Here is the code: https://github.com/TykTechnologies/graphql-go-tools/pull/203/files#diff-8ad913d5f2d2ad0b1886de4e5edb026559928b923de88a3cf2f8200102beefd1R298